src/css/brandColors.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
Can you explain why these are needed?

Reply:
oh sorry those were added during testing. forgot to remove it. updated